app/src/main/res/layout/feeditemlist_header.xml

```diff
@@ -126,6 +126,8 @@
             android:background="?android:attr/windowBackground"
             android:visibility="gone"
             android:gravity="center"
+            android:textColor="@color/accent_dark"
```

Comment:
This does not change the "filtered" text that the issue was about, but the "updates disabled" text. For my taste, making the font size bigger is a bit too much - it is pretty distracting to users who actually know that they want to filter.

Also, please use `?attr/colorAccent`. Otherwise, it will always use the accent color of the dark theme, which does not have a good contrast in the light theme.